src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -272,6 +273,9 @@ class NonlinearSolverParameters : public dataRepository::Group
 
   /// Flag to specify whether subcycling is allowed or not in sequential schemes
   integer m_subcyclingOption;
+
+  /// Value used to make sure that residual normalizers are not too small when computing residual norm
+  real64 m_minNormalizer = 1e-12;
```

Comment:
which residual was giving us issues?

Reply:
I am not 100% solid about this specific option (if this is the right solution) but what I observed for flow residual in hydrafracture cases - since the fracture volume is so tiny, the residual can go crazy because of normalization, so maybe better just to check the absolute value... more for experimenting for now.